webapp/src/containers/Rhs/projectDetails/index.tsx

```diff
@@ -37,7 +37,10 @@ const ProjectDetails = memo((projectDetails: ProjectDetails) => {
     });
     const [subscriptionList, setSubscriptionList] = useState<SubscriptionDetails[]>([]);
     const [showAllSubscriptions, setShowAllSubscriptions] = useState(false);
-    const [filter, setFilter] = useState(SubscriptionFilterCreatedBy.me);
+    const [filter, setFilter] = useState<SubscriptionFilters>({
+        createdBy: subscriptionFilters.createdBy.me,
+        serviceType: subscriptionFilters.serviceType.boards,
```

Comment:
Are we showing only `boards` subscription by default?

Reply:
flow changed now